engine/apps/api/serializers/alert.py

```diff
@@ -50,17 +54,22 @@ class Meta:
     def get_render_for_web(self, obj):
         return AlertFieldsCacheSerializerMixin.get_or_set_web_template_field(
             obj,
-            "render_for_web",
+            AlertFieldsCacheSerializerMixin.RENDER_FOR_WEB_FIELD_NAME,
             AlertWebRenderer,
         )
 
 
 class AlertRawSerializer(serializers.ModelSerializer):
     id = serializers.CharField(read_only=True, source="public_primary_key")
+    raw_request_data = serializers.SerializerMethodField()
 
     class Meta:
         model = Alert
         fields = [
             "id",
             "raw_request_data",
         ]
+
+    def get_raw_request_data(self, obj):
+        # TODO:
```

Comment:
do we want to show something more useful than just an empty object in this case?